spec/system/candidate_interface/candidate_entering_personal_details_spec.rb

```diff
@@ -72,6 +72,22 @@ def then_i_can_check_my_answers
     expect(page).to have_content 'Name'
     expect(page).to have_content 'Lando Calrissian'
     expect(page).to have_content 'Change'
+
+    expect(page).to have_content t('application_form.personal_details.date_of_birth.label')
+    expect(page).to have_content '6 April 1937'
+    expect(page).to have_content 'Change'
+
+    expect(page).to have_content t('application_form.personal_details.nationality.label')
+    expect(page).to have_content 'British and American'
+    expect(page).to have_content 'Change'
+
+    expect(page).to have_content t('application_form.personal_details.english_main_language.label')
+    expect(page).to have_content 'Yes'
+    expect(page).to have_content 'Change'
+
+    expect(page).to have_content t('application_form.personal_details.english_main_language_details.label')
+    expect(page).to have_content "I'm great at Galactic Basic so English is a piece of cake"
+    expect(page).to have_content 'Change'
```

Comment:
Just to note that this is the 5th time we're testing that the page contains the word "Change" 😉